osquery/tables/networking/posix/interfaces.cpp

```diff
@@ -127,13 +129,13 @@ void genDetailsFromAddr(const struct ifaddrs* addr, QueryData& results) {
       }
 
       if (ioctl(fd, SIOCGIFFLAGS, &ifr) >= 0) {
-        r["flags"] = INTEGER(static_cast<size_t>(ifr.ifr_flags));
+        flags |= static_cast<size_t>(ifr.ifr_flags);
       }
       close(fd);
     }
 
     // Flags populated by sysfs are more reliable.
```

Comment:
Please update this comment to reflect that only some flags are fetched from sysfs.